maven/archetypes/vrealize/pom.xml

```diff
@@ -1,38 +1,38 @@
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
```

Comment:
I am wondering whether we need this at all - isn't it overlapping with the mixed archetype? https://github.com/vmware/build-tools-for-vmware-aria/tree/main/maven/archetypes/mixed/src/main/resources/archetype-resources

Maybe an alternative would be to replace vra subproject here with vra-ng combined with actions + xml

Reply:
Hmm, but this is for `vrealize` not `vra`?

Reply:
Discussed offline, the current state is that after the merge the archetype contains only vro folder which has the same contents as mixed-project archetype (xml + js). This will be handled as separate issue.